CHANGELOG.rst

```diff
@@ -40,6 +40,8 @@ Fixed
   interactive learning
 - Pinned `networkx~=2.3.0` to fix visualization in `rasa interactive` and Rasa X
 
+- Fixed rasa init showing traceback error when user do Keyboard Interrupt before choose the path 
+
```

Comment:
This changelog post should go into the master section if you are merging to master (L28). However if you want it to go earlier into a bug fix, feel free to open it agains our patch branch, `1.4.x`!